pkg/apis/registries/v1alpha1/repository_types.go

```diff
@@ -27,6 +36,12 @@ type RepositorySpec struct {
 	// name of the harbor instance the repository is created for
 	ParentInstance corev1.LocalObjectReference `json:"parentInstance"`
 
+	// +optional
+	CountLimit int `json:"countLimit"`
```

Comment:
Is it necessary to differentiate between "not set" and "0", here?

Reply:
No, Harbor seems to set an unlimited value for the project either when setting the value to '0' or when omitting it entirely.